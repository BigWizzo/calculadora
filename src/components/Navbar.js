import React from 'react';
import { Link } from 'react-router-dom';

const Navbar = () => (
  <>
    <div className="nav-wrapper flex">
      <div className="logo">Math Magician</div>
      <div className="nav flex">
        <Link to="/" className="nav-link">Home </Link>
        <Link to="/calculator" className="nav-link">Calculator </Link>
        <Link to="/quote" className="nav-link">Quote </Link>
      </div>
    </div>
  </>
);

export default Navbar;
