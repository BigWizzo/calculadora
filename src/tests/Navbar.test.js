import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Navbar from '../components/Navbar';

test('should render the Home Page', () => {
  const component = renderer.create(
    <BrowserRouter>
      <Navbar />
    </BrowserRouter>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
