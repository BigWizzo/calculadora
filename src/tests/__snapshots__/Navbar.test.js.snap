// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Home Page 1`] = `
<div
  className="nav-wrapper flex"
>
  <div
    className="logo"
  >
    Math Magician
  </div>
  <div
    className="nav flex"
  >
    <a
      className="nav-link"
      href="/"
      onClick={[Function]}
    >
      Home 
    </a>
    <a
      className="nav-link"
      href="/calculator"
      onClick={[Function]}
    >
      Calculator 
    </a>
    <a
      className="nav-link"
      href="/quote"
      onClick={[Function]}
    >
      Quote 
    </a>
  </div>
</div>
`;
